app/assets/javascripts/application.js

```diff
@@ -12,12 +12,16 @@
 //
 //= require jquery
 //= require jquery_ujs
-//= require jquery.nestable
 
 // Required by Blacklight
 //= require jquery-ui
-//= require jquery.ui.nestedSortable
 //= require blacklight/blacklight
 //= require browse_everything
-//= require pop_help
-//= require input_form_attribute_support
+//= require mediaelement_rails
+//= require me-thumb-selector
+//= require me-add-to-playlist
```

Comment:
Should other mediaelement plugins be included here?  I didn't look through to see which we're really using but we have repos for mediaelement-qualityselector, mediaelement-skin-avalon, mediaelement-hd-toggle, mediaelement-title, mediaelement-logo-plugin.